oauth2_http/javatests/com/google/auth/oauth2/ImpersonatedCredentialsTest.java

```diff
@@ -585,13 +587,14 @@ void refreshAccessToken_GMT_dateParsedCorrectly() throws IOException, IllegalSta
                 // Set system timezone to GMT
                 Calendar.getInstance(TimeZone.getTimeZone("GMT")));
 
-    assertEquals(
-        c.getTime().toInstant().truncatedTo(ChronoUnit.SECONDS).toEpochMilli(),
-        targetCredentials.refreshAccessToken().getExpirationTimeMillis());
+    assertTrue(
```

Comment:
nit: assertEquals looks better here

Reply:
This was due to an issue with assertEquals reaching for the wrong overloaded method. Chatted in person and leaving as is.